Servers/UI/OJS.Servers.Ui/ClientApp/src/common/types.ts

```diff
@@ -211,6 +211,31 @@ interface IIndexProblemsType {
     contestId: number;
 }
 
+interface IIndexContestCategoriesType {
+    id: number;
+    name: string;
+    parent: string;
+    parentId: number;
+    isDeleted: boolean;
+    isVisible: boolean;
+    orderBy: number;
+    modifiedOn: Date;
+    createdOn: Date;
+    deletedOn: Date;
+}
+
+interface IContestCategoryAdministration {
+    id: number;
+    name: string;
+    parent: string;
+    parentId: number;
+    isDeleted: boolean;
+    isVisible: boolean;
+    orderBy: number;
+    deletedOn: Date | null;
```

Comment:
Why does this type have a null possibility but the type for IIndexContestCategoriesType  does not?